File: Amortization/Entities/AmortizationCls.cs
using Amortization.EnumTypes;
using System;
using System.Collections.Generic;

namespace Amortization.Entities
{
    public class AmortizationCls
    {
        public AmortizationCls()
        {
            Fees = new List<FeeCls>();
            AmortizationTypesValues = new AmortizationTypesValues { AmortizationType = (int)AmortizationTypes.FrenchType, AmortizationTypeValue = "Amortización Francesa" } ;
        }
        public Double TotalDebt { get; set; }
        public Double Interest { get; set; }
        public Double FeeValue { get; set; }
        public int NumberOfFee { get; set; }
        public AmortizationTypesValues AmortizationTypesValues { get; set; }
        public DateTime InitialDate { get; set; }
        public DateTime FinalDate { get; set; }
        public List<FeeCls> Fees { get; set; }
    }
}

File: Amortization/App/Amtz/src/app/data/models/AmortizationCls.ts
import { FeeCls } from "./FeeCls";
import { AmortizationTypesValues } from "./AmortizationTypesValues";

export class AmortizationCls {
    UserId:string;
    Order:number;
    AmortizationName:string;
    TotalDebt: number;
    Interest: number;
    FeeValue: number;
    NumberOfFee: number;
    AmortizationType: AmortizationTypesValues;
    InitialDate: Date;
    FinalDate: Date;
    Fees: Array<FeeCls>;
}